Fix apr_hash_first() doxygen syntax.
http://svn.apache.org/viewvc?view=revision&revision=829490

Doxgen would generate html file with full Userland build path in name
for this header.

--- include/apr_hash.h	2009/10/25 03:52:40	829489
+++ include/apr_hash.h	2009/10/25 04:12:21	829490
@@ -122,16 +122,15 @@
  * @param p The pool to allocate the apr_hash_index_t iterator. If this
  *          pool is NULL, then an internal, non-thread-safe iterator is used.
  * @param ht The hash table
+ * @return The iteration state
  * @remark  There is no restriction on adding or deleting hash entries during
  * an iteration (although the results may be unpredictable unless all you do
  * is delete the current entry) and multiple iterations can be in
  * progress at the same time.
- */
-/**
- * @example
  *
- * <PRE>
- * 
+ * @par Example:
+ *
+ * @code
  * int sum_values(apr_pool_t *p, apr_hash_t *ht)
  * {
  *     apr_hash_index_t *hi;
@@ -143,7 +142,7 @@
  *     }
  *     return sum;
  * }
- * </PRE>
+ * @endcode
  */
 APR_DECLARE(apr_hash_index_t *) apr_hash_first(apr_pool_t *p, apr_hash_t *ht);
 
